### PR TITLE
added Task60 PageFactory

### PR DESCRIPTION
**Pull request by ChristinaAstashkevich**

### CODE REVIEW

File: src/main/java/page/LoginPage.java [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+public class LoginPage {
+    private WebDriver driver = WebBrowser.getInstance();
+
+    public LoginPage(WebDriver driver) {
+        this.driver = driver;
+        PageFactory.initElements(driver, this);
+    }
+
+    @FindBy(css = "#mailbox__login")
```
Comment (EugenBorisik) [inline src/main/java/page/LoginPage.java]: use "id"

File: src/main/java/page/LoginPage.java [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    private WebElement loginField;
+
+    @FindBy(css = "#mailbox__password")
```
Comment (EugenBorisik) [inline src/main/java/page/LoginPage.java]: use "id"

File: src/main/java/page/LoginPage.java [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+    private WebElement passwordField;
+
+    @FindBy(css = "#mailbox__auth__button")
```
Comment (EugenBorisik) [inline src/main/java/page/LoginPage.java]: use "id"

**Review by EugenBorisik — APPROVED** [review]

File: src/main/java/page/LoginPage.java (hunk 1 above)
Comment (ChristinaAstashkevich) [reply src/main/java/page/LoginPage.java]: fixed

File: src/main/java/page/LoginPage.java (hunk 2 above)
Comment (ChristinaAstashkevich) [reply src/main/java/page/LoginPage.java]: fixed

File: src/main/java/page/LoginPage.java (hunk 3 above)
Comment (ChristinaAstashkevich) [reply src/main/java/page/LoginPage.java]: fixed

**Pull request merged**